style.css

```
@@ -226,42 +243,35 @@ nav  ul  li:hover {
 	flex-wrap: wrap;
 }
 
-#trablahos div.works figure{
```

```suggestion
#trabalhos .works figure
``` 
O nome da id `trabalhos` está escrito errado. 

Dica: não necessariamente você precisa incluir um `div.works`. Automaticamente usando um `.words` você já está atribuindo o estilo a essa classe.